### DISCUSSION TITLE
categorize on the main page

### DISCUSSION THREAD

**Discussion by hackitz** [linuxserver/Heimdall#799] [Ideas]
pulled from redit. hopefully this gets some upvotes. 

I was planning for 2.6.0 to rename the current tags to folders and have a new tagging system where they are all listed at the top and always visible, so you can click on a tag at any time and see the icons for that tag rather than going into a folder and having to come out again.

**Comment by KodeStar** [comment]
Now implemented in v2.6.0

In settings there is a new option called Treat Tags As with the options Folders, Tags and Categories.

**Folders**: This is the same as the current functionality.
**Tags**: This displays all the pinned items on the dashboard and has a list of tags at the top, clicking on a tag will only show items with that tag.
**Categories**: This displays pinned items in columns grouped by the tag.

<img width="1834" alt="Screenshot 2024-02-18 at 21 01 21" src="https://github.com/linuxserver/Heimdall/assets/5698566/a6f5f8f6-5b30-4a90-91d1-fdae6b6f8069">
<img width="1834" alt="Screenshot 2024-02-18 at 21 01 11" src="https://github.com/linuxserver/Heimdall/assets/5698566/4af43a78-5511-4093-a6d8-3148696675aa">
<img width="1834" alt="Screenshot 2024-02-18 at 21 02 02" src="https://github.com/linuxserver/Heimdall/assets/5698566/a144011e-ffd9-45fc-b1cd-dbd528b022a0">
